xblock/scorable.py

```diff
@@ -0,0 +1,140 @@
+"""
+Scorable.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from collections import namedtuple
+import logging
+
+
+log = logging.getLogger(__name__)
+
+
+Score = namedtuple('Score', ['earned', 'total'])
```

Comment:
Per @sanfordstudent's suggestion, can we be explicit and consistent with these names?  So: `raw_earned, raw_possible` instead of `earned, total`?  This is the terminology I introduced in https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/graders.py#L47.